src/hal/hal.cpp

```diff
@@ -184,6 +191,13 @@ static void hal_spi_init () {
     SPI.begin();
 }
 
+#if (defined(CFG_sx1261_radio) || defined(CFG_sx1262_radio))
+bit_t is_busy() {
```

Comment:
`is_busy()` should not be a global -- or should be `hal_is_radio_busy()` if it needs to be global for the driver.